cpp/merge_differential_and_full_marc_updates.sh

```diff
@@ -5,7 +5,7 @@ set -o errexit -o nounset -o history -o histexpand
 no_problems_found=1
 function SendEmail {
     if [[ $no_problems_found -eq 0 ]]; then
-        send_email --recipients="$email_address" --subject="$0 passed on $(hostname)" --message-body="No problems were encountered."
+        send_email --priority=very_low --recipients="$email_address" --subject="$0 passed on $(hostname)" --message-body="No problems were encountered."
```

Comment:
Ditto!

Reply:
This priority wasnt dropped, is was added, since all other success messages without problems are sent with low priority (on cpp side).